mesa/experimental/jupyter_viz.py

```diff
@@ -201,6 +206,12 @@ def on_value_play(change):
             playing=playing.value,
             on_playing=playing.set,
         )
+        widgets.IntText(
```

Comment:
Instead of `widgets.IntText`, maybe try `solara.Markdown` instead.

Also, your code means that the widget is created from scratch every time `on_value_play` is called. A Solara widget should only be created once, and then is updated.

Reply:
@rht thanks for the feedback. I didn't see an obvious Solara output field so I followed the other widget examples; I had it set to disabled/read-only, but I agree a real output would be better and I can see now that markdown is feasible.

I've pushed an update that replaces `widgets.IntText` with `solara.Markdown` and tested locally, it is updating the number when I play through the simulation.